common/src/main/java/org/opensearch/sql/common/utils/StringUtils.java

```diff
@@ -108,6 +108,48 @@ public static String format(final String format, Object... args) {
     return String.format(Locale.ROOT, format, args);
   }
 
+  /**
+   * Converts sql wildcard character % and _ to * and ?.
+   * @param text string to be converted
+   * @return converted string
+   */
+  public static String convertSqlWildcardToLucene(String text) {
```

Comment:
why put it in common module instead of openseach module?

Reply:
Will move this. Thanks!

Reply:
Fixed in https://github.com/opensearch-project/sql/commit/a31c9707b7b0a924fb0bd4f8e9600c9214885eae